lib/UnoCore/Backends/CIL/CIL.uxl

```diff
@@ -31,8 +31,7 @@
     <Template Name="Uno.Native">
         <Require Assembly="@(PACKAGE_DIR)/prebuilt/Uno.Native/Uno.Native.dll" />
         <Require UnmanagedLibrary="@(PACKAGE_DIR)/prebuilt/Uno.Native/libuAsset.dylib" Condition="HOST_MAC" />
-        <Require UnmanagedLibrary.x86="@(PACKAGE_DIR)/prebuilt/Uno.Native/x86/uAsset.dll" Condition="HOST_WIN32" />
-        <Require UnmanagedLibrary.x64="@(PACKAGE_DIR)/prebuilt/Uno.Native/x64/uAsset.dll" Condition="HOST_WIN32" />
+        <Require UnmanagedLibrary="@(PACKAGE_DIR)/prebuilt/Uno.Native/uAsset.dll" Condition="HOST_WIN32" />
```

Comment:
Don't we need both x86 and x64?

Reply:
Windows 32-bit support has been broken for a long time and it doesn't look like it's missed since no one have mentioned it. So I dropped the old x86 DLLs while repackaging to save some bandwidth/avoid some noise...